angular/app.component.js

```diff
@@ -0,0 +1,64 @@
+function Item(purchase, price, done) {
+    this.purchase = purchase;
+    this.price = price;
+    this.done = done;
+}
+ 
+var AppComponent = ng.core.Component({
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['ng' is not defined.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5238370)